elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -25,68 +29,63 @@ $LOCAL-VARIABLES: (
+  margin: pfe-local(Margin, $region: tab, $fallback: 0 0 -1px);
 
   // Padding
-  padding: #{pfe-local(PaddingTop, $region: tab)} #{pfe-local(PaddingRight, $region: tab)} #{pfe-local(PaddingBottom, $region: tab)} #{pfe-local(PaddingLeft, $region: tab)};
+  padding: pfe-local(PaddingTop, $region: tab) pfe-local(PaddingRight, $region: tab) pfe-local(PaddingBottom, $region: tab) pfe-local(PaddingLeft, $region: tab);
 
   // Border
-  border-top:     #{pfe-local($cssvar: BorderTop, $region: tab, $fallback: #{pfe-local($cssvar: BorderWidth, $region: tab, $fallback: #{pfe-var(ui--border-width)})} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor, transparent, $region: tab)} )};
-  border-right:   #{pfe-local($cssvar: BorderRight, $region: tab, $fallback: #{pfe-local($cssvar: BorderWidth, $region: tab, $fallback: #{pfe-var(ui--border-width)})} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor, transparent, $region: tab)} )};
-  border-bottom:  #{pfe-local(BorderBottom, 0, $region: tab)};
-  border-left:    #{pfe-local($cssvar: BorderLeft, $region: tab, $fallback: #{pfe-local($cssvar: BorderWidth, $region: tab, $fallback: #{pfe-var(ui--border-width)})} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor, transparent, $region: tab)} )};
+  border-top:     pfe-local(BorderTop, $region: tab, $fallback: pfe-local(BorderWidth, $region: tab, $fallback: pfe-var(ui--border-width)) pfe-var(ui--border-style) pfe-local(BorderColor, transparent, $region: tab));
+  border-right:   pfe-local(BorderRight, $region: tab, $fallback: pfe-local(BorderWidth, $region: tab, $fallback: pfe-var(ui--border-width)) pfe-var(ui--border-style) pfe-local(BorderColor, transparent, $region: tab));
+  border-bottom:  pfe-local(BorderBottom, 0, $region: tab);
+  border-left:    pfe-local(BorderLeft, $region: tab, $fallback: pfe-local(BorderWidth, $region: tab, $fallback: pfe-var(ui--border-width)) pfe-var(ui--border-style) pfe-local(BorderColor, transparent, $region: tab));
 
   // Inner style
-  background-color: #{pfe-local(BackgroundColor)};
-  color: #{pfe-local(Color)};
+  background-color: pfe-local(BackgroundColor);
+  color: pfe-local(Color);  // @todo replace with pfe(text-muted) in Kendall's color story
 
   cursor: pointer;
-  text-align: #{pfe-local(TextAlign, center, $region: tab)};
-  
+  text-align: pfe-local(TextAlign, center, $region: tab);
+
   // Exposing this for customization
-  text-transform: #{pfe-local($cssvar: TextTransform, $region: tab, $fallback: none)};
-  font-family:  #{pfe-var(font-family)};
-  font-weight:  #{pfe-var(font-weight--normal)};
-  font-size:    #{pfe-local(FontSize, $region: tab)};
-    
+  text-transform: pfe-local(TextTransform, $region: tab, $fallback: none);
+  font-family:  pfe-var(font-family);
+  font-weight:  pfe-var(font-weight--normal);
+  font-size:    pfe-local(FontSize, $region: tab);
+
   .pfe-tab > * {
-    color: inherit !important;
     font-size: inherit;
```

Comment:
i still see this on the sandbox
![image](https://user-images.githubusercontent.com/456863/88398505-7da3e100-cd93-11ea-8321-6283841236c0.png)